### PR TITLE
Book sugestions

### PR DESCRIPTION
**Pull request by Drocheg**
Adds book sugestion model

### CODE REVIEW

**Comment by Drocheg** [comment]
https://trello.com/c/Fz8Ja2Ug/29-crear-modelo-de-booksuggestion

File: app/models/book_sugestion.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class BookSugestion < ApplicationRecord
+  validates :author, :title, :link, presence: true
+
+  has_one :user, dependent: :destroy
```
Comment (Gmarn) [inline app/models/book_sugestion.rb]: Are you sure you want a  `dependent: :destroy`? 
[Here](https://guides.rubyonrails.org/association_basics.html) some help.

File: db/migrate/20180910185811_create_book_sugestions.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  def change
+    create_table :book_sugestions do |t|
+      t.string :editorial
+      t.decimal :price
+      t.string :author, null: false
+      t.string :title, null: false
+      t.string :link, null: false
+      t.string :publisher
+      t.string :year
+      t.references :user, foreign_key: true
```
Comment (Gmarn) [inline db/migrate/20180910185811_create_book_sugestions.rb:11]: Were you able to create a boo suggestion without an user?

**Review by Gmarn — CHANGES_REQUESTED** [review]

File: db/migrate/20180910185811_create_book_sugestions.rb (hunk 2 above)
Comment (Drocheg) [reply db/migrate/20180910185811_create_book_sugestions.rb:11]: In rails console, yes.

**Review by Gmarn — APPROVED** [review]

**Pull request merged**